docs/getting-started-with-django.asciidoc

```diff
@@ -0,0 +1,299 @@
+[[get-started-with-django]]
+== Get started with Django
+
+Getting Elastic APM set up for your Django project is easy, and there are various ways you can tweak it to fit to your needs.
+
+[[installation]]
+=== Installation
+
+Install the Elastic APM module using pip:
+
+[source,bash]
+----
+$ pip install elastic-apm
+----
+
+or add it to your project's `requirements.txt` file.
+
+NOTE: If you use Django with uwsgi, make sure to
+http://uwsgi-docs.readthedocs.org/en/latest/Options.html#enable-threads[enable
+threads].
+
+[[setup]]
+=== Setup
+
+Set up the Elastic APM module in Django with these two steps:
+
+1.  Add `elasticapm.contrib.django` to `INSTALLED_APPS` in your settings:
+
+[source,python]
+----
+INSTALLED_APPS = (
+   # ...
+   'elasticapm.contrib.django',
+)
+----
+
+1.  Choose an app name, and set the secret token if needed.
+
+[source,python]
+----
+ELASTIC_APM = {
+   'APP_NAME': '<APP-NAME>',
+   'SECRET_TOKEN': '<SECRET-TOKEN>',
+}
+----
+
+or as environment variables:
+
+[source,shell]
+----
+ELASTIC_APM_APP_NAME=<APP-NAME>
+ELASTIC_APM_SECRET_TOKEN=<SECRET-TOKEN>
+----
+
+You now have basic error logging set up, and everything resulting in a 500 HTTP status code will be reported to the APM Server.
+
+[[performance-metrics]]
+=== Performance Metrics
+
+To send performance metrics to Elastic APM, simply add our tracing middleware to your `MIDDLEWARES` settings:
+
+[source,python]
+----
+MIDDLEWARES = (
+    'elasticapm.contrib.django.middleware.TracingMiddleware',
+    # ...
+)
+----
+
+NOTE: For Django 1.8, you can add the same middleware to `MIDDLEWARE_CLASSES`.
+
+Make sure that it is the first middleware in the list.
+
+[[instrumenting-custom-python-code]]
+==== Instrumenting custom Python code
+
+To gain further insights into the performance of your code, please see
+<<./custom-instrumentation.asciidoc#instrumenting-custom-code, instrumenting custom code>>.
```

Comment:
This should work, but normally you should be able to just use the reference. @dedemorton ?